app/javascript/lib/validation/validate-file-content.js

```diff
@@ -425,6 +425,9 @@ async function parseFile(file, fileType, fileOptions={}, sizeProps={}) {
     // get any unhandled or deliberate short-circuits
     if (error instanceof ParseException) {
       parseResult.issues.push(['error', error.key, error.message])
+    } else if (error instanceof TypeError) {
+      const msg = 'File cannot be uploaded in it\'s current state. Please reach out for assistance'
+      parseResult.issues.push(['error', 'type-error', msg])
```

Comment:
Blocker: let's classify this as a kind of unhandled parse error, to make related analytics easier and more extensible.

```suggestion
      parseResult.issues.push(['error', 'parsed:unhandled:js-type', msg])
```

Reply:
I'm didn't use 'unhandled' because isn't this being handled by having it output a user-readable error message?

Reply:
I also think since 'type-error' is the actual error type it is useful to have as the error type in the mixpanel tracking. Does "js-type" have a meaning I am missing though? I might not be aware of what the significance is there and am open to using that since we do track the explicit error type elsewhere

Reply:
Thanks for helping me understand the original rationale!  Here's the rationale behind my suggestions.

**Why start with `parse:unhandled`?**
`parse:unhandled` errors do show users a readable error message, although it's generated by a built-in JS error object.  See e.g. https://mixpanel.com/s/3T3eal: the user saw the (admittedly cryptic) error message: "The requested file could not be read, typically due to permission problems that have occurred after a reference to a file was acquired."  It turns out that's the generic message for a certain JavaScript error object that's built into browser APIs.  

I'm unaware of a prior definition for `parse:unhandled`, but I'd suggest we define it as "when an error occurs that is not due to a specific validation issue in their file, and where we do not provide users guidance on how to fix it themselves".  Here, we can't parse their file, and the current error message doesn't indicate how the user can fix it without contacting us.

**Why include `js-type`?**
Simply `type-error` strikes me as easy to misinterpret.  If I'm looking at [CSFV errors by type](https://mixpanel.com/s/3PstMY), I think I'd be more likely to confuse an error typed as `type-error` with one of our validation errors, e.g. something like a group-vs-numeric annotation type validation error, or a validation error where a column header says content is numeric but the value isn't actually numeric.

Having `js-type` seems likelier to help the analyst intuit it's a _JavaScript_ TypeError, not a validation error like a format error or content error.  I initially thought having `-error` was redundant -- it's already classified as an error -- but I see your point that it helps map things.  

Maybe `parse:unhandled:js-typeerror` or `parse:unhandled:typeerror` would be best?  I've updated my suggestion to hopefully incorporate your rationale.

In any case, I appreciate you outlining your reasoning.  If my case above isn't persuasive, that's OK, I no longer consider this blocking!